snap/snapcraft.yaml

```diff
@@ -54,9 +54,15 @@ parts:
         snapcraftctl build
         TRIPLET_PATH="$SNAPCRAFT_PART_INSTALL/usr/lib/$(gcc -print-multiarch)"
         LIBSODIUM=$(readlink -n $TRIPLET_PATH/libsodium.so.18)
+        # Remove so the link can be recreated on re-builds
+        [ -f "$TRIPLET_PATH/libsodium.so" ] && rm "$TRIPLET_PATH/libsodium.so"
```

Comment:
Just `rm -f "$TRIPLET_PATH/libsodium.so"` is simpler and results in the same thing.